data/human/transport missions.txt

```diff
@@ -3570,3 +3570,87 @@ mission "Hauler VI cargo prototype 3"
 			`The Testament enters the planet's atmosphere at high speed and lands somewhat uncontrolled near a waste processing plant. The cargo hold is glowing red hot and some areas in the cargo hold appear to be on fire.`
 			`	Veronica disembarks, walks over to you, and apologizes for the difficult trip. She explains that the cooling systems failed, causing the sushi calamity you experienced during the multiple legs of your journey.`
 			`	You are paid <payment> and she asks you not to talk about this escort job to anybody. You leave the docks quickly in order to escape the horrible smell of overheated fish that is coming from Veronica and her ship.`
+
+
+
+mission "The New Rigellian Plague"
+	name "Transport Vaccination Clinic"
+	description "Transport a team of nurses and vaccine supplies to <destination>. Payment will be 100,000 credits."
+	deadline
+	cargo "vaccine" 1
+	passengers 2
+	to offer
+		random < 100
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	destination
+		distance 3 5
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	on offer
+		conversation
+			`The spaceport entry is practically at a standstill today, jammed with a winding line of people standing around irritably. You overhear one person grumble to another "Why'd they have to pick here to do this?" Their companion responds with a grin "You know as well as I do that if they did it somewhere out of the way, no-one would get the vaccination. They keep telling us that it is only a matter of time until it comes back, but how likely is that?"`
+			`	As the line moves forward, you can see a banner stating "Red Cross Vaccination Clinic - Free Rigellian Plague Vaccinations." The nurses are surprisingly fast, vaccinating people and giving them a card with information at a fast clip. Before you can even make a decision, you are at the front of the line and a nurse is standing in front of you scanning your ID.`
+			`	"Good morning Captain! I see your medical records indicate you haven't received the vaccine yet," she pauses, skimming a few lines on her scanner. "Ah, good. You aren't allergic to anything in this vaccine, and haven't had any reactions to medicines in the past. This vaccine is designed to boost your immune system against three variants of the Rigellian Plague. It won't guarantee your immunity, but it will significantly improve your resistance to it. In which arm would you prefer to receive the vaccine?"`
```

Comment:
```suggestion
			`	"Good morning Captain! I see your medical records indicate you haven't received the vaccine yet," she pauses, skimming a few lines on her scanner. "Ah, good. You aren't allergic to anything in this vaccine, and haven't had any reactions to medications in the past. This vaccine is designed to boost your immune system against three variants of the Rigellian Plague. It won't guarantee your immunity, but it will significantly improve your resistance to it. In which arm would you prefer to receive the vaccine?"`
```